examples/extending_models/models/models.gen.go

```diff
@@ -7,13 +7,12 @@ import (
 	"database/sql"
 	"database/sql/driver"
 	"fmt"
-	"strings"
-	"sync"
-
 	"github.com/ethanpailes/pgtypes"
 	"github.com/opendoor/pggen"
 	"github.com/opendoor/pggen/include"
 	"github.com/opendoor/pggen/unstable"
+	"strings"
```

Comment:
This looks funny. I think we should be running `go fmt` over these automatically. Any idea how these stdlib packages ended up here?

Reply:
not sure, `go fmt` doesn't seem to move them back up